ui/app/templates/components/secret-link.hbs

```diff
@@ -1,10 +1,3 @@
-<LinkTo
-  @route={{this.link.route}}
-  @models={{this.link.models}}
-  @query={{this.query}}
-  @disabled={{@disabled}}
-  {{on "click" this.onLinkClick}}
```

Comment:
This was only being used in the `SecretVersionMenu` component to close the menu but since a new route is transitioned to the menu is torn down making it unnecessary.